platform/viewer/src/routes/Mode/Mode.tsx

```diff
@@ -75,7 +79,8 @@ function defaultRouteInit(
     // hanging protocol in the mode configuration
     hangingProtocolService.run(
       { studies, activeStudy, displaySets },
-      hangingProtocolId
+      hangingProtocolId,
+      { stageIndex: hangingProtocolStageIndex || 0 }
```

Comment:
This should not pass in 0 if it isn't specified in the URL.  There is matching rules to decide the initial protocol stage, and specifically passing it in will disable some of those.